src/popup/components/TransferReview.vue

```diff
@@ -21,13 +21,13 @@
     <DetailsItem
       v-if="isAddressUrl"
       data-cy="review-tip-url"
-      class="tip-url"
+      class="tip-url details-item"
       :label="$t('pages.send.receivingUrl')"
       :value="transferData.address"
     />
     <DetailsItem
       v-else
-      data-cy="review-recipient"
+      data-cy="review-recipient details-item"
```

Comment:
cypress might not support multiple names

Reply:
I think Odo is just made a mistake.